data/data-pipeline/data_pipeline/etl/sources/cdc_svi_index/etl.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger
+from data_pipeline.score import field_names
+from data_pipeline.config import settings
+
+
+logger = get_module_logger(__name__)
+
+
+class CDCSVIINDEX(ExtractTransformLoad):
+    """CDC SVI Index class ingests 2018 dataset located
+    here: https://www.atsdr.cdc.gov/placeandhealth/svi/index.html
+    Please see the README in this module for further details.
+    """
+
+    def __init__(self):
+        self.OUTPUT_PATH = self.DATA_PATH / "dataset" / "cdc_svi_index"
+
+        self.CDC_SVI_INDEX_S3_URL = (
+            settings.AWS_JUSTICE40_DATASOURCES_URL + "/SVI2018_US.zip"
+        )
+
+        self.CDC_RPL_THEMES_THRSHOLD = 0.90
+
+        self.CDC_SVI_INDEX_ZIP_FILE_DIR = self.TMP_PATH / "cdc_svi_index"
+
+        self.CDC_SVI_INDEX_TRACTS_FIPS_CODE = "FIPS"
+
+        self.COVARIATE_COLUMNS = [
+            "AREA_SQMI",  # Tract area in square miles
+            "E_TOTPOP",  # Population estimate, 2014-2018 ACS S0601_C01_001 E
+            "E_HU",  # Housing units estimate, DP04_0001E
+            "E_HH",  # Households estimate, DP02_0001E
+            # E_PCI has fewer rows than other variables - joined to Census 2016 tracts. Contains null cells (i.e. -999).
+            "E_PCI",  # Per capita income estimate, 2014-2018 ACS, B19301_001E
+            "E_AGE65",  # Persons aged 65 and older estimate, 2014-2018 ACS, S0101_C01_030 E
+            "E_DISABL",  # Civilian noninstitutionalized population with a disability estimate, 2014 -2018 ACS, DP02_0071E
+            "E_SNGPNT",  # Single parent household with children under 18 estimate MOE, 2014 -2018 ACS, (DP02_0007E + DP02_0009E)
+            "E_CROWD",  # At household level (occupied housing units), more people than rooms estimate for 2014 -2018 ACS, (DP04_0078E + DP04_0079E)
+        ]
+
+        self.COLUMNS_TO_KEEP = [
+            self.GEOID_TRACT_FIELD_NAME,
+            field_names.CDC_SVI_INDEX_SE_THEME_PERCENTILE_FIELD,  # SE prefix references "socioeconomic"
+            field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_PERCENTILE_COMPOSITION_FIELD,
+            field_names.CDC_SVI_INDEX_LANGUAGE_THEME_PERCENTILE_FIELD,
+            field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_PERCENTILE_FIELD,
+            field_names.CDC_SVI_INDEX_RPL_THEMES_OVERALL_PERCENTILE_FIELD,
+            # Sum of series fields
+            field_names.CDC_SVI_INDEX_SE_THEME_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_LANGUAGE_THEME_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_THEMES_OVERALL_SOS_FIELD,
+            # Derived columns
+            field_names.CDC_SVI_INDEX_THEMES_OVERALL_TOTAL_FIELD,
+            field_names.CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY,
+        ] + self.COVARIATE_COLUMNS
```

Comment:
Do we need COVARIATE_COLUMNS in the columns to keep?

Reply:
Addressed your comment [here](https://github.com/usds/justice40-tool/pull/1096#discussion_r783349210)